db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180920200522) do
+ActiveRecord::Schema.define(version: 20181003135506) do
```

Comment:
Should `db/schema.rb` be included in the `.gitignore` file like we do for the [ManageIQ/manageiq](https://github.com/ManageIQ/manageiq/blob/master/.gitignore#L56) repo?

Reply:
I personally think it should be https://stackoverflow.com/questions/6450263/what-is-the-right-approach-to-deal-with-rails-db-schema-rb-file-in-git

It is nice to know db schema changes and track them in git. It's also easier to see those changes via `schema.rb` than backwards engineering migration files.